documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1022 @@
+
+
+* **at_hash**
+
+  OPTIONAL. Access Token hash value. Its value is the base64url encoding of the left-most half of the hash of the octets of the ASCII representation of the access_token value, where the hash algorithm used is the hash algorithm used in the alg Header Parameter of the ID Token's JOSE Header. For instance, if the alg is RS256, hash the access_token value with SHA-256, then take the left-most 128 bits and base64url-encode them. The `at_hash` value is a case-sensitive string.
+
+
+ID Tokens MAY contain other Claims. Any Claims used that are not understood MUST be ignored.
+
+
+## Client Authentication
+
+This section defines a set of Client Authentication methods that are used by Clients to authenticate to the Authorization Server when using the Token or Backchannel Authorization Endpoint.
+
+These Client Authentication methods are:
+
+* **_private_key_jwt_**
+
+  RECOMMENDED. Clients that have registered a public key sign a JWT using that key. The Client authenticates in accordance with JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants [OAuth.JWT] and Assertion Framework for OAuth 2.0 Client Authentication and Authorization Grants [OAuth.Assertions]. At least the Authorization Server MUST support JWS-signed Object with the RS256 algorithm. The JWT MUST contain the following REQUIRED Claim Values and MAY contain the following OPTIONAL Claim Values:
+
+    * **iss**
+
+      REQUIRED. Issuer. This MUST contain the `client_id` of the OAuth Client.
+
+    * **sub**
+
+      REQUIRED. Subject. This MUST contain the `client_id` of the OAuth Client.
+
+    * **aud**
+
+      REQUIRED. Audience. The aud (audience) Claim. Value that identifies the Authorization Server as an intended audience. The Authorization Server MUST verify that it is an intended audience for the token. The Audience MUST be the URL of the Authorization Server's Token Endpoint.
+
+    * **jti**
+
+      REQUIRED. JWT ID. A unique identifier for the token, which can be used to prevent reuse of the token. These tokens MUST only be used once, unless conditions for reuse were negotiated between the parties; any such negotiation is beyond the scope of this specification.
+
+    * **exp**
+
+      REQUIRED. Expiration time on or after which the JWT MUST NOT be accepted for processing.
+
+    * **iat**
+
+      OPTIONAL. Time at which the JWT was issued.
+
+The JWT MAY contain other Claims. Any Claims used that are not understood MUST be ignored.
+
+The authentication token MUST be sent as the value of the [OAuth.Assertions](https://www.rfc-editor.org/info/rfc7521) `client_assertion` parameter.
+
+The value of the [OAuth.Assertions](https://www.rfc-editor.org/info/rfc7521) `client_assertion_type` parameter MUST be `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`, per [OAuth.JWT](https://www.rfc-editor.org/info/rfc7523).
+
+
+* **_client_secret_basic_**
```

Comment:
`client_secret_basic` should be removed as an option. `private_key_jwt` should be mandatory.

Reply:
The reference to client_secret_basic as a supported authentication method is removed, as agreed at the last meeting.